src/Library/demos/TextView/main.blp

```diff
@@ -0,0 +1,74 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "TextView";
+  description: _("A widget that enables text-editing");
+
+  Box {
+    orientation: vertical;
+    halign: center;
+
+    ScrolledWindow {
+    height-request: 180;
+    width-request: 600;
+
+        TextView textview {
+          bottom-margin: 12;
+          left-margin: 12;
+          right-margin: 12;
+          top-margin: 12;
+          margin-bottom: 12;
+          editable: true;
+          cursor-visible: true;
+          wrap-mode: char;
+          }
+        }
+
```

Comment:
Let's put this `ScrolledWindow` in a `Gtk.Frame`, then move `TextView {margin-bottom: 12; }` to `Frame { margin-bottom: 12; }`.

This boxes up the TextView nicely, and it makes it visible that the text continues below the viewport.